app/controllers/meetings_controller.rb

```diff
@@ -22,15 +22,19 @@ def show
       end
     else
       @comment = Comment.new
-      @comments = Comment.where(:commented_on => @meeting.id, :comment_type => "meeting").all.order("created_at DESC")
+      @comments = Comment.where(commented_on: @meeting.id, comment_type: 'meeting').all.order('created_at DESC')
       @no_hide_page = true
     end
   end
 
   def comment
-    @comment = Comment.new(:comment_type => params[:comment_type], :commented_on => params[:commented_on], :comment_by => params[:comment_by], :comment => params[:comment], :visibility => 'all')
+    @comment = Comment.new(comment_type: params[:comment_type], commented_on: params[:commented_on], comment_by: params[:comment_by], comment: params[:comment], visibility: 'all')
 
-    if !@comment.save
+    # TODO: it is quite unclear what the expected outcome is here. If nothing
+    #       more should be done, then a return statement should be added. If
+    #       the follow work should be performed, then the responce actions
+    #       should be grouped at the end of the method
+    unless @comment.save
```

Comment:
The expected outcome is to render `{ no_save: true }` to indicate that the comment was able to be saved. Otherwise, create notifications.